lib/python-debugger-view.coffee

```diff
@@ -72,6 +72,7 @@ class PythonDebuggerView extends View
     @stopApp() if @backendDebugger
     @debuggedFileArgs = @getInputArguments()
     console.log @debuggedFileArgs
+    @debuggedFileName = @getCurrentFilePath()
```

Comment:
Should the following `if` be removed and replaced with a call to `askForPaths()` then?

Reply:
The change is adding the last line, which set the file to debug as the current active file.

The if is to check whether we need to stop the debugging (if the debugger is already running). This is the original code. It has nothing to do with the change.

Reply:
What I'm saying is that, because you set `debuggedFileName`, the next `if` will never be active (`pathsNotSet()` will always return `false`).

Reply:
Sorry, I misunderstood your comment.

Reply:
It is up to you incorporate the changes. I am happy, as long as it works. :-)